index.html

```diff
@@ -253,10 +253,11 @@ <h2>So, what's the verdict?</h2>
 <p>At a baseline level, this works. HTTP Signatures attached to ActivityPub inbox delivery requests can effectively verify the actor - or at least the server - who sent them. However, these signatures are ephemeral. They only authenticate HTTP requests, not long-lived data. We'd often like to authenticate objects and activities outside of inbox delivery requests, eg during <a href="https://www.w3.org/TR/activitypub/#inbox-forwarding">inbox forwarding</a>, or after they were initially created. HTTP Signatures can't do this.</p>
-<p>Some fediverse projects like Mastodon use <a href="https://web.archive.org/web/20170923124140/https://w3c-dvcg.github.io/ld-signatures/">LD Signatures 1.0</a> for those purposes instead, which works, but isn't widely supported. Even Mastodon's support itself is limited and discouraged. <a href="https://docs.joinmastodon.org/spec/security/#ld">From their docs</a>:</p>
+<p>Some fediverse projects based on the Mastodon API use <a href="https://web.archive.org/web/20170923124140/https://w3c-dvcg.github.io/ld-signatures/">LD Signatures 1.0</a> for object-authentication purposes instead, which works, but is unevenly supported within the Mastodon-API ecosystem. Even Mastodon's ongoing support for it is limited and deprecated, as, like Cavage-12, the final normative recommendation is not backwards-compatible. <a href="https://docs.joinmastodon.org/spec/security/#ld">From Mastodon's docs</a>:</p>
 <blockquote>
 Mastodon’s current implementation of LD Signatures is outdated...Furthermore, the LD Signatures specification as a whole has been superseded by [[[vc-data-integrity]]], which is largely incompatible with the earlier LD Signature spec. For this reason, it is not advised to implement support for LD Signatures.
 </blockquote>
+<p>It is worth noting that there is a long-standing, iterated Fediverse Enhancement Proposal, <a href="https://codeberg.org/fediverse/fep/src/branch/main/fep/8b32/fep-8b32.md">FEP-8b32</a>, which specifies how to use the superceding normative reference for [[[vc-data-integrity]]] to authenticate objects independent of server-to-server trust. This FEP has been implemented by <a href="https://codeberg.org/fediverse/fep/src/branch/main/fep/8b32/fep-8b32.md#implementations">6 distinct AP implementations,</a> including multiple "nomadic identity"-based servers and clients, one additional client-signing implementation (mitra), and the Fedify library which powers Ghost's ActivityPub implementation and hollo. Activity- and object-authentication serves a different purpose than server-to-server authentication, though, so recommendations on upgrading to [[[vc-data-integrity]]] or dropping that class of authentication altogether are out of scope of this report.</p>
```

Comment:
This is great! I may trim this down a bit, both for length and to remove the specific implementation details, so that we don't have to keep updating them over time, but in general this is a good addition!